tests/test_api.py

```diff
@@ -88,27 +88,21 @@ def test_with():
     assert not exists(run_dir)
 
 
-@patch.object(MAGICC, 'is_initialised')
-def test_with_init_once(mock_is_initialised):
-    with patch.object(MAGICC, 'init') as mock_init:
-        # Check that init not called if is_initialised is true
-        mock_is_initialised.return_value = True
-        with MAGICC():
-            pass
-        mock_init.assert_not_called()
-
-        mock_is_initialised.return_value = False
-        with MAGICC():
-            pass
-        mock_init.assert_called_once()
-
-
-def test_is_initalised():
-    p = MAGICC()
-    assert not p.is_initialised()
-    p.init()
-    assert p.is_initialised()
-
-    # test a duplicated object
-    p2 = MAGICC(p.root_dir)
-    assert p2.is_initialised()
+def test_create_copy_only_once():
+    with pytest.raises(FileExistsError):
+        m = MAGICC()
+        # Create copy.
```

Comment:
I would put the mock back in here or at least have another unit test which uses the mocking framework